### ISSUE TITLE
web-wa-sqlite walkthrough ends in a broken state.

### ISSUE THREAD

**Issue by CitrusFruits** [electric-sql/electric#799]
I'm trying to give ElectricSQL a try, but can't even get through the [most basic tutorial](https://github.com/electric-sql/electric/tree/main/examples/web-wa-sqlite) without hitting a snag.

## Steps to Reproduce
1. `git clone https://github.com/electric-sql/electric`
2. `cd electric/examples/web-wa-sqlite`
3. `yarn backend:up`
4. `yarn db:migrate`

## Expected Result
The command completes successfully without errors.

## Actual Result
The following is output
```
❯ yarn client:generate

yarn run v1.22.21
$ yarn electric:check && npx electric-sql generate --service http://localhost:5133 --proxy postgresql://prisma:proxy_password@localhost:65432/electric
$ node ./check-electric-is-running.js
Error: Command failed: npx prisma db pull --schema=".electric_migrations_tmp_URmywS/prisma/schema.prisma"
Error:
P4001 The introspected database was empty:

prisma db pull could not create any models in your schema.prisma file and you will not be able to generate Prisma Client with the yarn prisma generate command.

To fix this, you have two options:

- manually create a table in your database.
- make sure the database connection URL inside the datasource block in schema.prisma points to a database that is not empty (it must contain at least one table).

Then you can run yarn prisma db pull again.


    at ChildProcess.exithandler (node:child_process:402:12)
    at ChildProcess.emit (node:events:525:35)
    at maybeClose (node:internal/child_process:1100:16)
    at Socket.<anonymous> (node:internal/child_process:458:11)
    at Socket.emit (node:events:513:28)
    at Pipe.<anonymous> (node:net:301:12) {
  code: 1,
  killed: false,
  signal: null,
  cmd: 'npx prisma db pull --schema=".electric_migrations_tmp_URmywS/prisma/schema.prisma"'
}
generate command failed: Introspection script exited with error code: 1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by samwillis** [comment]
Hey @CitrusFruits,

Thanks for the report.

It looks like the yarn lock file is broken for that example resulting in a missing dependancy. This meant that the migrate command failed to complete.

I've opened a PR to fix this (#800) which will get merged soon.

Until then, if you delete the `electric/examples/web-wa-sqlite/yarn.lock` file in the checkout, and then run `yarn` to install all the dependancies. Everything should then work.

**Comment by balegas** [comment]
#800 is merged. Thanks for reporting the issue.
